### ISSUE TITLE
Issues trying to get Nest Protect to work

### ISSUE THREAD

**Issue by ashwind85** [iMicknl/ha-nest-protect#235]
### The problem

New guy to HA here. I downloaded this from HACS and went to "Devices and Integration"  to add the Nest Protect integration to my HA set-up. It then asked me to "link your google account, authorize your account" and asked for a token."

When I click the hyperlink, it says that m access is blocked and that Nest's request is invalid.

I'm not sure what i'm doing wrong. I just want to be able to integrate my Nest  Protect alarms in my house to HA using this. Any ELI5 help would be appreciated. 

### What version of this integration (ha-nest-protect) has the issue?

0.3.12

### What version of Home Assistant Core has the issue?

2023.6.1

### Home Assistant log

<details><summary>Logs</summary>
```
Copy/paste any log here, between the starting and ending backticks (`)
```

</details>

**Comment by iMicknl** [comment]
Sorry, this is a bit unclear at the moment. You need the latest beta release (https://github.com/iMicknl/ha-nest-protect/releases), you can choose in HACS. You need to specifically choose this when you click on reinstall.

See documentation for after installation here:
https://github.com/iMicknl/ha-nest-protect/tree/v0.4.0b1#retrieving-issue_token-and-cookies

**Comment by ashwind85** [comment]
Thanks this did the trick!
